src/emulator/io/CMakeLists.txt

```diff
@@ -11,5 +11,5 @@ add_library(
 )
 
 target_include_directories(io PUBLIC include)
-target_link_libraries(io PUBLIC miniz vita-headers dirent)
+target_link_libraries(io PUBLIC host miniz vita-headers dirent)
```

Comment:
This creates a circular dependency, because host already depends on io (HostState contains IOState.)

I'm surprised CMake allows it actually.

Reply:
I removed host stuff, but I still need rtc utility functions.
I could either pull I'll pull rtc into its own project, or move the functions to util, I think I'm gonna do the former.